packages/fluentui/react-northstar/src/components/Loader/Loader.tsx

```diff
@@ -142,9 +141,12 @@ export const Loader = (React.forwardRef<HTMLDivElement, LoaderProps>((props, ref
     return () => clearTimeout(delayTimer.current);
   }, [delay]);
 
-  const svgElement = Box.create(svg, {
-    defaultProps: () => ({ className: loaderSlotClassNames.svg, styles: resolvedStyles.svg }),
```

Comment:
Let's also modify `loaderSlotClassNames` to delete `svg` from there

Reply:
Thank you, I removed the svg key from the loaderSlotClassNames object